app/views/shared/_lawyer.html.erb

```diff
@@ -56,6 +56,15 @@
 
         <div>
           <%= form.submit 'Save', class: 'btn btn-primary' %>
+          <% if @lawyer.approved? %>
+            <%= link_to "Resend Confirmation Email", approve_admin_lawyer_path, method: :put, params: { id: lawyer.id }, class: 'btn btn-success' %>
```

Comment:
This functionality should only be available to admin, as well as the addition/viewing of private comments above. This partial can be seen by both lawyers (ie their profile) and admins. So I think you'll want to wrap all the admin specific logic in ````<% if current_user.admin? %>```` so that lawyers don't see anything they shouldn't be.

Let me know if you need help/clarification with this